### DISCUSSION TITLE
How do I extract .bin files from an .ab file?

### DISCUSSION THREAD

**Discussion by GamerXYZ2** [AssetRipper/AssetRipper#636] [Q&A]
I have an .ab file I know contains .bin files. I can even see the English text in these .bin files in text form if I use a program to decompress the .ab file without extracting it, when I open it in a hex editor. Yet when I extract the files from this .ab file with AssetRipper (I tried the original file and the decompressed file, both do the same), I don't get these .bin files. Instead I get tiny 1kb .asset and .asset.meta files that don't contain any of the information the .bin files are supposed to hold. What am I doing wrong?

**Comment by ds5678** [comment]
I'll be posting a new release in a couple days when I return home. Until then, you're welcome to use the alpha build.
